packages/dx-react-chart/src/plugins/spline-series.test.tsx

```diff
@@ -48,24 +48,28 @@ describe('Spline series', () => {
     },
   };
 
-  it('should render path', () => {
-    const tree = mount((
-      <PluginHost>
-        {pluginDepsToComponents(defaultDeps)}
+  it('ad', () => {
 
-        <SplineSeries
-          {...defaultProps as any}
-        />
-      </PluginHost>
-    ));
-
-    expect(tree.find(SeriesComponent).props()).toEqual({
-      pointComponent: undefined,
-      index: 1,
-      coordinates: coords,
-      color: 'color',
-      getAnimatedStyle: undefined,
-      scales: { xScale: 'arg-scale', yScale: 'val-scale' },
-    });
   });
+
+  // it('should render path', () => {
```

Comment:
Does SplineSeries chart work?

Reply:
Yes, it does. But the test doesn't work.